### PR TITLE
manual-install: add IMAGE_TAG to omit separate latest-arm64.yml

### PR DESCRIPTION
**Pull request by ManOki**
Add an additional parameter IMAGE_TAG to configure the docker-compose.yml image attributes, i.e. the architecture amd64 or arm64.

This PR is especially useful if PR #1586 is also accepted and optional services can be omitted if they are not available on certain architectures like clamav on arm64.

### CODE REVIEW

**Review by szaimen — COMMENTED** [review]
Hm... Not sure about this as clamav is not available on arm64...

**Comment by szaimen** [comment]
> Not sure about this as clamav is not available on arm64...

Due to this I would honestly rather keep it sepeate. WDYT @ManOki ? :)

**Comment by ManOki** [comment]
@szaimen I think this is a good PR :) otherwise I would have not created it.

But of course without #1586, users will get an error for clamav on arm64 and are not able to start their containers so the other PR is a hard dependency. 
Another advantage is that you may use other image tags as well like `develop` or `beta`
https://hub.docker.com/r/nextcloud/aio-nextcloud/tags
https://hub.docker.com/r/nextcloud/aio-collabora/tags

**Comment by szaimen** [comment]
All right! Then please add a note to the readme stating that clamav is not supported on arm64. Also we will need to adjust the sctipt for this as well since it will recreate the latest-arm64 file otherwise.

**Comment by ManOki** [comment]
There is already a [hint for clamav](https://github.com/ManOki/nextcloud-all-in-one/blob/docker_image_tag/manual-install/readme.md?plain=1#L24) and an updated version of [update-yaml.sh](https://github.com/ManOki/nextcloud-all-in-one/blob/docker_image_tag/manual-install/update-yaml.sh#L126). I did not test the complete script as I don't know/have the runtime environment.

**Comment by szaimen** [comment]
This needs a rebase now due to conflicts...

**Comment by szaimen** [comment]
Do you need help with the rebase? :)

File: manual-install/update-yaml.sh [hunk 1]
```diff
@@ -128,12 +130,6 @@ networks:
...
 
 cat containers.yml > latest.yml
-sed -i '/image:/s/$/:latest/' latest.yml
-
-cat containers.yml > latest-arm64.yml
-sed -i '/image:/s/$/:latest-arm64/' latest-arm64.yml
-sed -i '/  nextcloud-aio-clamav:/,/^$/d' latest-arm64.yml
-sed -i '/nextcloud[-_]aio[-_]clamav/d' latest-arm64.yml
-sed -i '/CLAMAV_ENABLED/d' latest-arm64.yml
+sed -i '/image:/s/$/:${IMAGE_TAG}/' latest.yml
```
Comment (szaimen) [inline manual-install/update-yaml.sh:133]:
```suggestion
sed -i '/image:/s/$/:\${IMAGE_TAG}/' latest.yml
```
You can check if this fixes SC

File: manual-install/sample.conf [hunk 2]
```diff
@@ -1,8 +1,9 @@
+IMAGE_TAG=latest          # Version of docker images, should be latest or latest-arm. Note: latest-arm64 has no clamav support
```
Comment (szaimen) [inline manual-install/sample.conf:1]:
```suggestion
IMAGE_TAG=latest          # Version of docker images, should be latest or latest-arm64. Note: latest-arm64 has no clamav support
```

File: manual-install/update-yaml.sh [hunk 3]
```diff
@@ -128,12 +130,6 @@ networks:
...
 
 cat containers.yml > latest.yml
-sed -i '/image:/s/$/:latest/' latest.yml
-
-cat containers.yml > latest-arm64.yml
-sed -i '/image:/s/$/:latest-arm64/' latest-arm64.yml
-sed -i '/  nextcloud-aio-clamav:/,/^$/d' latest-arm64.yml
-sed -i '/nextcloud[-_]aio[-_]clamav/d' latest-arm64.yml
-sed -i '/CLAMAV_ENABLED/d' latest-arm64.yml
+sed -i '/image:/s/$/:\${IMAGE_TAG}/' latest.yml
```
Comment (szaimen) [inline manual-install/update-yaml.sh:133]: Another attempt:
```suggestion
sed -i "/image:/s/$/:\${IMAGE_TAG}/" latest.yml
```

File: manual-install/update-yaml.sh [hunk 4]
```diff
@@ -57,6 +58,7 @@ do
 done
 
 sed -i 's|_ENABLED=|_ENABLED=no          # Setting this to "yes" enables the option in Nextcloud automatically.|' sample.conf
+sed -i 's|CLAMAV_ENABLED=no|_ENABLED=no          # Setting this to "yes" enables the option in Nextcloud automatically. Note: latest-arm64 has no clamav support|' sample.conf
```
Comment (szaimen) [inline manual-install/update-yaml.sh:61]:
```suggestion
sed -i 's|CLAMAV_ENABLED=no|CLAMAV_ENABLED=no          # Setting this to "yes" enables the option in Nextcloud automatically. Note: latest-arm64 has no clamav support|' sample.conf
```

**Review by szaimen — APPROVED** [review]
Thanks, LGTM! :)

**Comment by ManOki** [comment]
Thank you for your detailed review and fixes.

**Comment by szaimen** [comment]
No problem! Thank you for your contributions! :)

**Pull request merged**